config.yml

```diff
@@ -22,6 +22,16 @@ blacklist:
    # Unaccessible metrics (not enough privilege)
    - java:lang:memorypool:.*usagethreshold.*
 
+   # Unaccessible metrics Java 11 (not enough privilege)
```

Comment:
I'm unsure this makes sense to disable that in the default configuration file as I am expecting most of the user of this exporter to be running against a Cassandra 4.0 and using Java 8 - it makes no sense for them to disable these metrics, and I'm expecting they represent most of the users for at least a bit of time.

Maybe comment these values to make discovery easier for Java11/Cassandra 4.0 ?

Maybe maintainers have a point of view on that?